### PR TITLE
Site redesign 2024

### PR DESCRIPTION
**Pull request by GLDuval**

### CODE REVIEW

File: src/views/index.ts [hunk 1]
```diff
@@ -28,14 +20,6 @@ const routes: RouteRecordRaw[] = [
       title: i18n.global.t('page_team'),
     },
   },
-  {
```
Comment (patates-cipsi418) [inline src/views/index.ts:31]: Idk if we should open an issue to add those pages later

File: src/views/index.ts [hunk 2]
```diff
@@ -12,14 +12,6 @@ const routes: RouteRecordRaw[] = [
       title: i18n.global.t('page_home'),
     },
   },
-  {
```
Comment (patates-cipsi418) [inline src/views/index.ts:15]: Idk if we should open an issue to add those pages later

File: index.html [hunk 3]
```diff
@@ -41,4 +18,4 @@
   <script type="module" src="/src/main.ts"></script>
 </body>
 
-</html>
+</html>
```
Comment (patates-cipsi418) [inline index.html:21]: miss an ending return line

File: src/components/SponsorCarouselComponent.vue [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+import AutoPlay from 'embla-carousel-autoplay';
+import aeets from '@clubcapra/assets/media/partners/aeets.png';
+import monday from '@clubcapra/assets/media/partners/monday.png';
+import odrive from '@clubcapra/assets/media/partners/odrive.png';
+import opnor from '@clubcapra/assets/media/partners/opnor.png';
+import stelpro from '@clubcapra/assets/media/partners/stelpro.png';
+
+const [emblaRef] = emblaCarouselVue({}, [AutoPlay({ delay: 5000 })]);
+
+const sponsors = [
```
Comment (patates-cipsi418) [inline src/components/SponsorCarouselComponent.vue:12]: This should be get from a store

File: src/views/PartnersView.vue
Comment (patates-cipsi418) [inline src/views/PartnersView.vue:1]: Should open an issue to have a store with the patrners

**Review by patates-cipsi418 — CHANGES_REQUESTED** [review]

File: src/views/TeamView.vue [hunk 5]
```diff
@@ -1,21 +1,17 @@
 <script setup lang="ts">
```
Comment (patates-cipsi418) [inline src/views/TeamView.vue:1]: Nicolas is no longer in admin

**Review by patates-cipsi418 — CHANGES_REQUESTED** [review]

**Comment by GLDuval** [comment]
@patates-cipsi418 Should be good now, I moved the partners and team members into new files in a `data` folder to make the views cleaner.

**Comment by GLDuval** [comment]
Once approved we just need to make sure that we squash the commits and merge

**Review by patates-cipsi418 — APPROVED** [review]

**Pull request merged**